app/views/order_items/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render "shared/header" %>
```

Comment:
Does every view (minus partials) render the shared header? If so, could it be put in the application layout instead?